doc/authorization-config.md

```diff
@@ -494,6 +532,14 @@ One of the following forms may be used as the value for an `allow` entry:
  of `xyz.domain.org` and a request URL of
  `"http://my-host:8080/the/path/xyz"`, however, would not be considered a 
  match for the entry.
+ 
+ * A map with either a `certname` or `extensions` key. The former behaves
+   exactly as a standalone string. The latter is used to specify a set of ssl
+   extensions to match. Each key specified in the extensions map must appear in
+   the request and the values are matched exactly (no wildcarding). Extensions
+   in the request not explicitly listed here are ignored. Note that a given
+   extension map won't match if not *all* keys match. If you want an "or"
+   relationship between extensions, split them into different extension maps.
```

Comment:
Somewhere in this description, I'd mention wording like..

1) Each key included in the extension map must have a matching extension in the request certificate.
2) The value associated with each extension key may be a string or a list of strings.  If the extension value is a string, it must be an exact match for the corresponding extension's value in the request certificate in order for the entry to be a match.  If the extension value is a list, one or more of the items in the list must match the corresponding extension's value in the request certificate.

Where this is a little hard to explain, I think a couple of examples of what do / do not constitute a match would be helpful to include here.